### ISSUE TITLE
Parsing error with interpolated expression

### ISSUE THREAD

**Issue by jpzwarte** [sasstools/sass-lint#321]
```left: calc(50% - #{$eb-hint-width / 2});```

This causes a parser error to be thrown: "Please check validity of the block starting from line #40"

Using sass-lint-1.3.1

**Comment by DanPurdy** [comment]
There's a problem with interpolation within the current version of the AST we use, it looks to be fixed in the new version though, once it's stable we'll be moving up to it.

Similar issue here #210

**Comment by micahgodbolt** [comment]
Also found in #313

**Comment by DanPurdy** [comment]
This is still causing an error in gonzales-pe 3.2.1 have logged an issue over there. You can find it [here](https://github.com/tonyganch/gonzales-pe/issues/115)

Will keep you updated

**Comment by stowball** [comment]
Just voicing a +1 as it happened to me as soon as I tried to implement sass-lint :(

**Comment by DanPurdy** [comment]
+1s here are kind of useless as we're blocked by a dependency, it would be better for you to report this on the gonzales-pe repository

**Comment by stowball** [comment]
Noted!

**Comment by clcpolevaulter** [comment]
hey @DanPurdy or @bgriffith, do you anticipate a release date for the gonzales 3.2.6 fixes

**Comment by bgriffith** [comment]
Hey @clcpolevaulter - we're just finishing off the remaining issues with the rules. I'd hope we could do a release by next week if not sooner.

**Comment by webgefrickel** [comment]
@bgriffith that would be very cool, since I am experiencing this issue as well, when trying to do sth. like:

```scss
#{$side}: $value;
```

**Comment by colin-marshall** [comment]
@webgefrickel good news, that was definitely fixed in 3.2.6. I put in an issue for it that was resolved a couple weeks back (tonyganch/gonzales-pe#140) so the forthcoming sass-lint 1.6.0 should resolve that.

**Comment by clcpolevaulter** [comment]
hey @DanPurdy or @bgriffith, we are really looking forward to these changes. Any updates on when the next release will be?

**Comment by colin-marshall** [comment]
@clcpolevaulter: @DanPurdy said earlier today that it will hopefully be ready by this weekend https://github.com/zurb/foundation-sites/issues/7620#issuecomment-188161798

**Comment by DanPurdy** [comment]
Sorry guys I'm just really swamped with work at the moment but I'll definitely be looking to have this last rule updated and done very soon..

**Comment by clcpolevaulter** [comment]
@DanPurdy @bgriffith is there a branch in progress on this release? I'd love to help if needed.

**Comment by DanPurdy** [comment]
@clcpolevaulter We're actually done and ready to release 1.6 really BUT the AST currently doesn't install on windows correctly so we can't really release it until that's sorted..

See the bottom of this issue https://github.com/tonyganch/gonzales-pe/issues/149 for more info

**Comment by EightArmCode** [comment]
Any word on this?

**Comment by DanPurdy** [comment]
@EightArmCode the majority of these issues should hopefully be fixed in our 1.6 update as it includes the latest version of the AST we've been using. I would expect it to come out in the next few days.. Hopefully..

**Comment by Regaddi** [comment]
@DanPurdy Any progress on this? Really looking forward to it! :smiley:

**Comment by bgriffith** [comment]
@jpzwarte, @Regaddi, @EightArmCode, @clcpolevaulter  - This is fixed in 1.6

**Comment by DanPurdy** [comment]
ping @colin-marshall too ^^
